doc/README.md

```diff
@@ -3,13 +3,13 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions (which is currently more than 100 GBs); depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core, the original bitcoin client, builds the network's backbone. It downloads and, by default, stores the entire history of bitcoin transactions (which is currently more than 160 GB). Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to over a day.
```

Comment:
I like the author's suggestion of a different first sentence, which could be integrated: "Bitcoin Core is the original and current reference implementation of the Bitcoin network" https://github.com/bitcoin/bitcoin/pull/13010#discussion_r184677201